webview-codicons-sample/media/buttons.js

```diff
@@ -0,0 +1,17 @@
+// @ts-ignore
+
+// This script will be run within the webview itself
+// It cannot access the main VS Code APIs directly.
+
```

Comment:
Let's get rid of the javascript for this example. You can hardcode some spinner example if we think they are interesting

Reply:
Agreed and removed for the sake of simplification. I moved the styling/interaction sample to the bottom of the page as it indeed is optional.

Reply:
Javascript files removed for the sake of simplicity. The styling and interaction sample moved to the bottom as it indeed is optional.